.gitignore

```diff
@@ -1,9 +1,16 @@
 .DS_Store
-
+*.swp
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
 deps/deps.jl
 docs/build
 
 Manifest.toml
+/src/core/constraint_jose.jl
+/src/core/ref_jose.jl
+/src/core/variable_jose.jl
+/src/prob/fs_mc_jose.jl
+/test/fs_mc_jose.jl
+/test/test_output.txt
+/test/test_output_no_pv.txt
```

Comment:
an easier way to do this would be to exclude files with a pattern like `*_jose.*` or `test_output*`